node/src/components/consensus/highway_core/state.rs

```diff
@@ -52,6 +52,9 @@ use tallies::Tallies;
 // endorsements again.
 pub(super) const TODO_ENDORSEMENT_EVIDENCE_DISABLED: bool = true;
 
+// Disables checking the limited naivete criterion for endorsements.
```

Comment:
Typo or what is "naivete criterion"?

Reply:
I checked the Highway protocol PDF and it should be "naivetY" I guess.

Reply:
You're right, we should make it consistent. My English dictionary has both "naivety" and "naïveté", though.